statsmodels/tsa/filters/filtertools.py

```diff
@@ -308,8 +308,7 @@ def convolution_filter(x, filt, nsides=2):
     return result
 
 
-#copied from sandbox.tsa.garch
```

Comment:
Format and leave the comment.  Should someone ever want to see history, tells them to look at this files history.

Reply:
more generally this is probably the strategy that should be used to move functions if/when ready.  A clean comment back to its original location so that someone could find the history, If they need to see what went wrong.